src/ora-dashboard/data-table/DataTable.jsx

```diff
@@ -1,67 +1,106 @@
 import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 import React, { useEffect } from 'react';
+import { Table } from '@edx/paragon';
 import { useDispatch } from 'react-redux';
 import { useParams } from 'react-router';
-import LoadingOrValue from '../../components/value-or-loading/ValueOrLoading';
 import messages from './messages';
 import { fetchOraReports } from '../data/thunks';
 import { oraDataShape } from '../../data/constants';
 
+/**
+ * Sort implementation for Paragon Table
+ * @param {any} firstElement
+ * @param {any} secondElement
+ * @param {string} key
+ * @param {string} direction
+ */
+const sort = function sort(firstElement, secondElement, key, direction) {
```

Comment:
I think naming the function twice is a bit redundant. 
```suggestion
function sort(firstElement, secondElement, key, direction) {
```

Reply:
Also this  seems like a utility function, so can probably moved to a common places and imported as such.